windows/src/unit-tests/keyboard-package-versions/KeyboardPackageVersionsTestSuite.dpr

```diff
@@ -107,7 +108,7 @@ begin
     runner.UseRTTI := True;
     //tell the runner how we will log things
     //Log to the console window
-    logger := TDUnitXConsoleLogger.Create(false); //true);
+    logger := TDUnitXConsoleLogger.Create(true);
```

Comment:
Just checking you wanted to leave this as `true`

Reply:
Yeah, I think that's fine. The `false` was there originally during development.